content/en/blog/_posts/2024-09-30-gateway-api-v1.2/index.md

```diff
@@ -0,0 +1,418 @@
+
+Gateway API for north-south traffic has always had the concept that declaring
+a Gateway causes the backing infrastructure (for example, a Service and a
+Deployment) to be created by the Gateway API controller. One complication with
+this approach is that sometimes it's important to label or annotate the
+infrastructure: for example, it's commonly necessary to use one of these
+mechanisms to tell a service mesh to incorporate the Gateway infrastructure.
+
+In 1.2.0, the Gateway `infrastructure` stanza moves to the Standard channel,
+allowing you to specify labels and annotations for the infrastructure created
+by the Gateway API controller. For example, the following Gateway
+configuration specifies both Linkerd and Istio injection, allowing the Gateway
+infrastructure to be incorporated into whichever service mesh you've
+installed:
+
+```
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: meshed-gateway
+  namespace: incoming
+spec:
+  gatewayClassName: meshed-gateway-class
+  listeners:
+  - name: http-listener
+    protocol: HTTP
+    port: 80
+  infrastructure:
+    labels:
+      istio-injection: enabled
+    annotations:
+      linkerd.io/inject: enabled
+```
+
+For more information, check out the [`infrastructure` API
+reference](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.GatewayInfrastructure).
+
+#### [Backend Protocol Support](https://gateway-api.sigs.k8s.io/geps/gep-1911/)
+
+Since Kubernetes 1.20, the Service and EndpointSlice resources have supported
+a stable `appProtocol`  field to allow users to specify the L7 protocol that
+Service supports. With the adoption of [KEP
+3726](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/3726-standard-application-protocols),
+Kubernetes now supports three new `appProtocol` values:
+
+- `kubernetes.io/h2c` - HTTP/2 over cleartext as described in
+[RFC7540](https://www.rfc-editor.org/rfc/rfc7540)
+- `kubernetes.io/ws` - WebSocket over cleartext as described in
+[RFC6445](https://www.rfc-editor.org/rfc/rfc6445)
+- `kubernetes.io/wss` - WebSocket over TLS as described in
+[RFC6445](https://www.rfc-editor.org/rfc/rfc6445)
+
+With Gateway API 1.2.0, support for honoring `appProtocol` is now Standard.
+For example, given the following Service:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: websocket-service
+  namespace: my-namespace
+spec:
+  selector:
+    app.kubernetes.io/name: websocket-app
+  ports:
+    - name: http
+      port: 80
+      targetPort: 9376
+      protocol: TCP
+      appProtocol: kubernetes.io/ws
+```
+
+then an HTTPRoute that includes this Service as a `backendRef` will
+automatically upgrade the connection to use WebSockets rather than assuming
+that the connection is pure HTTP.
+
+For more information, check out the
+[GEP-1911](https://gateway-api.sigs.k8s.io/geps/gep-1911/).
+
+### New additions to Experimental channel
```

Comment:
nit: this section feels like it falls under `## Graduations to the Standard Channel`, did you mean to have this as `## New Additions to Experimental Channel`?

Reply:
Looks like I was very inconsistent with heading levels in general, nice catch!